src/index.tsx

```diff
@@ -6,4 +6,5 @@ import type { Operation } from './impl/Operation';
 import type { Zone } from './Zone';
 import type { Campaign } from './Campaign';
 
-export { Condition, Action, AppEvent, Operation, Zone, Campaign, ExtoleImpl as Extole };
+export type { Condition, Action, AppEvent, Operation, Zone, Campaign };
+export { ExtoleImpl as Extole };
```

Comment:
Separate type exports from implementation exports so other tools know to ignore them.

Reply:
https://github.com/evanw/esbuild/issues/314#issuecomment-668401819

Reply:
I dont think builder bob is using esbuild but the maintainer calls out "Babel's TypeScript compiler" and I am pretty sure that is what builder bob uses under the hood.